### PR TITLE
Beta 2

### PR DESCRIPTION
**Pull request by mbouzi**
- changes for hompage
- mock - https://www.figma.com/file/VgUVW6kv30sjvIavWOObo4/Tardigrade.io-(Copy)?node-id=1882%3A2947

### CODE REVIEW

File: content/home/halfPrice/halfPrice.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+heading: "S3-compatible and Half the Price"
+col1Heading: "Storage"
+col1Image: "/img/tar-ico-cloud.png"
+col1Amount: "$10.00"
+col1Metric: "Per TB Per Month"
+col2Heading: "Bandwidth"
+col2Image: "/img/tar-ico-arrows-left-right.png"
+col2Amount: "$50.00"
```
Comment (jenlij) [inline content/home/halfPrice/halfPrice.md]: was this supposed to be $45?

File: content/home/milestones/heading/milestones.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
 title: "milestones-heading"
 heading: Roadmap Milestones to Production
-subheading: Tardigrade is currently in the Pioneer 1 phase of our beta. Those who sign up for our waitlist can also participate in our Beta ahead of production later this year.
----
\ No newline at end of file
+subheading: Tardigrade is currently in the Pioneer 2 phase of our beta. Those who sign up for our waitlist can also participate in our Beta ahead of production later this year.
```
Comment (jenlij) [inline content/home/milestones/heading/milestones.md:4]: is it later this year, or early next year? is there an official date we can give?

File: content/home/milestones/milestones/milestone3.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title : "milestone"
 heading : "Pioneer 1: V3 Network Beta"
-status : isCurrent
+status : isFuture
```
Comment (jenlij) [inline content/home/milestones/milestones/milestone3.md:4]: why is this isFuture? Shouldn't it be isPast or something?

File: content/home/usecases/data-tabs/tab2.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title : "usecase-tab"
 name : media
 heading : Media Content Storage
-desc : Store large volumes of video, photo, or audio media, including tape / physical alternatives.
+desc : Store large volumes of video, photo, or audio media, including tape/physical alternatives.
```
Comment (jenlij) [inline content/home/usecases/data-tabs/tab2.md:5]: what is tape/physical alternatives mean? just curious

File: content/home/usecases/usecases/usecase2.md [hunk 5]
```diff
@@ -2,6 +2,6 @@
 title: "usecases"
 name : media
 heading : Media Storage
-isActive : true
+isActive : false
```
Comment (jenlij) [inline content/home/usecases/usecases/usecase2.md:5]: what does this do?

File: content/home/halfPrice/halfPrice.md (hunk 1 above)
Comment (mbouzi) [reply content/home/halfPrice/halfPrice.md]: good catch!

File: content/home/milestones/heading/milestones.md (hunk 2 above)
Comment (mbouzi) [reply content/home/milestones/heading/milestones.md:4]: hmm Taylor reviewed and said all the copy was correct, but this is a good question

File: content/home/milestones/milestones/milestone3.md (hunk 3 above)
Comment (mbouzi) [reply content/home/milestones/milestones/milestone3.md:4]: I think this is the class that was given to both past and future milestones

File: content/home/usecases/data-tabs/tab2.md (hunk 4 above)
Comment (mbouzi) [reply content/home/usecases/data-tabs/tab2.md:5]: I have no idea lol but this was reviewed by Taylor today

File: content/home/usecases/usecases/usecase2.md (hunk 5 above)
Comment (mbouzi) [reply content/home/usecases/usecases/usecase2.md:5]: toggles active links in the "Tardigrade Cloud Object Storage Use Cases" section

File: content/home/milestones/milestones/milestone3.md (hunk 3 above)
Comment (jenlij) [reply content/home/milestones/milestones/milestone3.md:4]: ah ok haha

File: content/home/usecases/data-tabs/tab2.md (hunk 4 above)
Comment (jenlij) [reply content/home/usecases/data-tabs/tab2.md:5]: lol

**Pull request merged**